src/test/java/com/puppycrawl/tools/checkstyle/utils/CheckUtilTest.java

```diff
@@ -443,6 +443,28 @@ public void testParseClassNames() {
             .isEqualTo(expected);
     }
 
+    @Test
+    public void testIsPackageInfo() {
+        assertWithMessage("Should return true when is package info")
```

Comment:
We do not need pure UTs methods.
All should be tested by Input files.

If you need extra testing, please create files in our resources folder to let Checkstyle run on them and skip them.

Reply:
Please do not resolve my comments, I will do this. Reply "done".

Reply:
Done